selfdrive/car/toyota/carcontroller.py

```diff
@@ -91,14 +96,31 @@ def update(self, enabled, active, CS, frame, actuators, pcm_cancel_cmd, hud_aler
     if (frame % 3 == 0 and CS.CP.openpilotLongitudinalControl) or pcm_cancel_cmd:
       lead = lead or CS.out.vEgo < 12.    # at low speed we always assume the lead is present so ACC can be engaged
 
+      # record accelerator depression frame
+      if CS.out.gasPressed:
+        self.last_gas_press_frame = frame
+      # record standstill exit frame
+      if CS.pcm_acc_status == 7:
+        self.last_standstill_frame = frame
+      # record last frame when vego is 0
+      if CS.out.vEgo == 0:
+        self.last_zero_speed_frame = frame
+      # handle permit braking
+      if (CS.out.gasPressed or 1. / DT_CTRL > (frame - self.last_gas_press_frame)) \
+         or ((actuators.accel > - 1.95) and (2. / DT_CTRL > (frame - self.last_standstill_frame))) \
+         or (2. / DT_CTRL > (frame - self.last_zero_speed_frame)):
```

Comment:
So we can not brake for up to a second after pressing the gas? I'm not sure this is what we want exactly, and how it affects stop and go performance while openpilot is engaged.

Reply:
Yes I do, however I'm using the pedal only to get the car out of standstill, the pcm takes care of the acceleration afterwards, as seen in #23122

Reply:
Perhaps we could do something like if we're undershooting our speed and we don't want to brake, set the bit